tests/test_bevel.py

```diff
@@ -1,85 +1,55 @@
-#!/usr/bin/env python3
 import hashlib
 from pathlib import Path
+from tests.shared import hex_code_check
 
 from cycax.cycad import SheetMetal
 
 
-def fancy_edge(edge_loc):
-    sheet = SheetMetal(x_size=100, y_size=200, z_size=50, part_no="sheet_bevel")
 
-    sheet.beveled_edge(edge_type="round", side1="FRONT", side2="RIGHT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="FRONT", side2="LEFT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BACK", side2="LEFT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BACK", side2="RIGHT", size=3)
+def fancy_edge(edge_loc, edge_type:str, part_no:str):
+    sheet = SheetMetal(x_size=100, y_size=200, z_size=50, part_no=part_no)
 
-    sheet.beveled_edge(edge_type="round", side1="TOP", side2="FRONT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BOTTOM", side2="FRONT", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BACK", side2="TOP", size=3)
-    sheet.beveled_edge(edge_type="round", side1="BACK", side2="BOTTOM", size=3)
+    sheet.beveled_edge(edge_type=edge_type, side1="FRONT", side2="RIGHT", size=3)
```

Comment:
Is this all all the possible edge combinations.
Do you know itertools? Seems that https://docs.python.org/3/library/itertools.html#itertools.combinations would be much simpler.

Reply:
iteration is very cool. I'm used to having to write the method.